launcher/sdw_updater_gui/UpdaterApp.py

```diff
@@ -19,7 +32,7 @@ def __init__(self, parent=None):
         self.setupUi(self)
         self.clientOpenButton.setEnabled(False)
         self.clientOpenButton.hide()
-        self.clientOpenButton.clicked.connect(self.launch_securedrop_client)
+        self.clientOpenButton.clicked.connect(launch_securedrop_client)
```

Comment:
Since you've moved this as a function to L14, is the method in `UpdaterApp` class now dead code? https://github.com/freedomofpress/securedrop-workstation/blob/0cbe309ccecd0a00c140f5e58af8a588262b63b0/launcher/sdw_updater_gui/UpdaterApp.py#L176

Reply:
It is! deleted.